api/jobs.go

```diff
@@ -386,8 +386,7 @@ func (api *API) PutNumTasksHandler(w http.ResponseWriter, req *http.Request) {
 	// set eTag on ETag response header
 	dpresponse.SetETag(w, updatedETag)
 
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusOK)
+	w.WriteHeader(http.StatusNoContent)
```

Comment:
(question): why do you need a no content status here ?

Reply:
This is because status code is more explicit as no content is send in the response payload body

Reply:
ok, makes sense.